pages/styles/login.css

```diff
@@ -0,0 +1,35 @@
+.auth-form {
+  margin-top: 5rem;
+  height: 100%;
+}
+
+.auth-form form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* background-color: var(--color-primary); */
+  /* color: var(--color-basic); */
```

Comment:
Delete the commented code before pushing it to the dev

Reply:
Done :+1:

Reply:
Done :+1: